src/core/send.c

```diff
@@ -1018,6 +1018,45 @@ QuicSendGetNextStream(
     return NULL;
 }
 
+BOOLEAN
+CxPlatIsRouteReady(
+    _In_ QUIC_CONNECTION *Connection
+    )
+{
+    QUIC_PATH* Path = &Connection->Paths[0];
```

Comment:
I just realized after looking at the path challenge code, that this function must take `Path` as an argument, because the callers are operating on different paths. Not just `Paths[0]`.

Reply:
great catch, thanks